### ISSUE TITLE
Predictable specific script download

### ISSUE THREAD

**Issue by dougg0k** [undergroundwires/privacy.sexy#243]
Hi,

Is there a way to download a specific script on-the-fly in a predictable url?

So far, i've only seen download through clicking in the UI, and urls generated from it seems to be random uuids.

If not, can you add a way?

It would be useful since scripts can be randomly updated. If I have a automated script downloading the latest version of a specific script on execution, having this feature would certainly help.

**Comment by undergroundwires** [comment]
Hi, thanks for the idea. let us refine the idea further before implementation. Once migration to Vue 3 is done (#230), I can add [Vue router](https://v3.router.vuejs.org/) to handle URLs. I guess there should be different rendering for script-based URLs. I will add ID's to every script in upcoming releases so every script has unique identifier that's combination of operating system + partly GUID to support import/export #59. These will enable the functionality but we need to refine further how the UI will look like for single script. I think this issue is duplicate of #49 so I'd be happy if you could share your ideas about how UI/URL would look like. I.e. show script + revertcode + related docs? And how would share button look like?

**Comment by dougg0k** [comment]
I dont care much about UI, as long as there is a way, as each script having their own unique url regardless of updates in them. I think your idea there make sense, OS + unique identifier per script.
